external/fv3kube/fv3kube/config.py

```diff
@@ -157,13 +156,25 @@ def get_full_config(
 
 
 def c48_initial_conditions_overlay(
-    url: str, timestep: str, restart_categories: RestartCategoriesConfig = None
+    url: str,
+    timestep: str,
+    vertical_coordinate_file: str = DEFAULT_VERTICAL_COORDINATE_FILE,
+    restart_categories: RestartCategoriesConfig = None,
 ) -> Mapping:
     """An overlay containing initial conditions namelist settings
     """
     TIME_FMT = "%Y%m%d.%H%M%S"
     time = datetime.datetime.strptime(timestep, TIME_FMT)
     time_list = [time.year, time.month, time.day, time.hour, time.minute, time.second]
+
+    source_location, source_name = os.path.split(vertical_coordinate_file)
+
+    assert source_name != "", "Provided vertical coordinate file is a directory"
+
+    FV_CORE_ASSET = fv3config.get_asset_dict(
```

Comment:
nit: [based on PEP8 style](https://peps.python.org/pep-0008/#function-and-variable-names) we typically name variables (as opposed to constants) with all lowercase letters (be sure to update the name further down as well):
```suggestion
    fv_core_asset = fv3config.get_asset_dict(
```